### PR TITLE
Replace Red Hat and Noto Sans with Inter

### PR DESCRIPTION
**Pull request by roj1512**

### CODE REVIEW

File: site/docs/.vuepress/plugins/current-versions/index.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
```
[the remark this answers is not in the document]
Comment (roj1512) [reply site/docs/.vuepress/plugins/current-versions/index.json:1]: Seems like a flake, reverting.

[the remark this answers is not in the document]
Comment (KnorpelSenf) [reply site/docs/.vuepress/plugins/current-versions/index.json:1]: It should be gitignored, how did you even manage to commit this

File: site/docs/.vitepress/theme/style/_fonts.scss [hunk 2]
```diff
@@ -1,74 +1,18 @@
...
-  font-display: swap;
-  font-family: "Red Hat Display";
-  font-style: normal;
-  font-weight: 700;
-  src: url("/fonts/red-hat-display-v14-latin-ext_latin-700.woff2")
-    format("woff2");
-}
-/* noto-sans-display-700 - cyrillic-ext_cyrillic */
-@font-face {
-  font-display: swap;
```
Comment (KnorpelSenf) [inline site/docs/.vitepress/theme/style/_fonts.scss:12]: So we now use block not swap? Can you explain?

Comment (roj1512) [reply site/docs/.vitepress/theme/style/_fonts.scss:12]: No, we’re still using swap. This `font-display` thing is in another line now. See the diff below on the right side.

Comment (KnorpelSenf) [reply site/docs/.vitepress/theme/style/_fonts.scss:12]: I found it

File: site/docs/public/inter.css [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  font-style: normal;
+  font-weight: 100 900;
+  font-display: block;
+  src: url("/fonts/InterVariable.woff2") format("woff2");
+}
+@font-face {
+  font-family: InterVariable;
+  font-style: italic;
+  font-weight: 100 900;
+  font-display: block;
```
Comment (KnorpelSenf) [inline site/docs/public/inter.css:17]: So this is unused?

Comment (roj1512) [reply site/docs/public/inter.css:17]: Oh yes it is not used I should remove it

File: site/docs/.vuepress/plugins/current-versions/index.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (roj1512) [reply site/docs/.vuepress/plugins/current-versions/index.json]: It was the .vuepress directory checked out from a very old clone of main.

[the remark this answers is not in the document]
Comment (KnorpelSenf) [reply site/docs/.vuepress/plugins/current-versions/index.json]: I see

**Review by KnorpelSenf — APPROVED** [review]
LGTM

**Pull request merged**